gio/Gir.toml

```diff
@@ -916,6 +915,17 @@ status = "generate"
     # use strv
     manual = true
 
+[[object]]
+name = "Gio.FileMonitor"
+status = "generate"
+    [[object.function]]
+    name = "emit_event"
+        [[object.function.parameter]]
+        name = "other_file"
```

Comment:
this should be set as nullable upstream

Reply:
I triggered the process chain to get new GIR files & changes in this repo, you can rebase once that is over.